week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -23,47 +23,49 @@ PasswordValidationResult=  [false, false, false, false, true]
 */
 
 function validatePasswords(passwords) {
-
+  const check = passwords.filter(element => element.includes(element.length >= 5) && element === "A")
```

Comment:
to check the length of password you don't need to use of includes, and for check the other conditions you  need to use of regEx and test method